### PR TITLE
PAR Rodrigo and Jeremie

### PR DESCRIPTION
**Pull request by JeremieHercberg**
Some issues with the syntax on some questions

### CODE REVIEW

File: queries.md [hunk 1]
```diff
@@ -4,40 +4,88 @@
 
 ### 1. All the companies that it's name match 'Babelgum'. Retrieve only their `name` field.
 
+filter: {name: 'Babelgum'}
+project :{name: 1, \_id: 0}
```
Comment (ta-web-paris) [inline queries.md]: do
not
use
the
\

:skull:

File: queries.md [hunk 2]
```diff
@@ -4,40 +4,88 @@
...
 ### 10. Retrieve the 10 companies with more employees, order by the `number of employees`
 
+sort: {number_of_employees: -1}
+limit : 10
+
 ### 11. All the companies founded on the second semester of the year. Limit your search to 1000 companies.
 
+filter : {founded_month: {$gte:7}}
+
 ### 12. All the companies that have been 'deadpooled' after the third year.
```
Comment (ta-web-paris) [inline queries.md]: filter : {$where: "this.deadpooled_year - this.founded_year > 3"}

File: queries.md [hunk 3]
```diff
@@ -4,40 +4,88 @@
...
 
 ### 13. All the companies founded before 2000 that have and acquisition amount of more than 10.000.000
 
+filter :{$and:[ {founded_year: {$lt :2000}},{'acquisition.price_amount': {$gt: 10000000}}]}
+
 ### 14. All the companies that have been acquired after 2015, order by the acquisition amount, and retrieve only their `name` and `acquisiton` field.
```
Comment (ta-web-paris) [inline queries.md]: filter : {'acquisition.acquired_year': {$gt: 2015}}
project : {name: 1, acquisition: 1, _id: 0}
sort : {'acquisition.price_amount': 1}

**Comment by JeremieHercberg** [comment]
I think it was a typo because I don’t remember use it backslash...

**Pull request closed without merging**